src-input/duk_bi_date_windows.c

```diff
@@ -86,10 +86,15 @@ DUK_INTERNAL_DECL duk_int_t duk_bi_date_get_local_tzoffset_windows(duk_double_t
 	ft1.dwLowDateTime = tmp2.LowPart;
 	ft1.dwHighDateTime = tmp2.HighPart;
 	FileTimeToSystemTime((const FILETIME *) &ft1, &st2);
+#if defined(_DURANGO) || defined(_XBOX_ONE)
+	// XboxOne doesn't have 'SystemTimeToTzSpecificLocalTime' API
```

Comment:
I believe it leaves the final time in UTC, which honestly is probably not what you want. But there is no equivalent to SystemTimeToTzSpecificLocalTime in the Durango XDK, so that was the best I could come up with.  I thought it was better than returning zero.

Also, I'm guessing based on your other comment that I shouldn't have done a platform check in this code; that should be in duk_config.h.